src/js_native_api_v8.cc

```diff
@@ -2871,6 +2871,49 @@ napi_status napi_is_promise(napi_env env,
   return napi_clear_last_error(env);
 }
 
+napi_status napi_create_date(napi_env env,
+                             double time,
+                             napi_value* result) {
+  NAPI_PREAMBLE(env);
+  CHECK_ARG(env, result);
+
+  auto maybe_date = v8::Date::New(env->context(), time);
+  CHECK_MAYBE_EMPTY(env, maybe_date, napi_generic_failure);
+
+  auto v8_resolver = maybe_date.ToLocalChecked();
```

Comment:
This is copied from the `Promise` equivalent, right? There’s no resolver here :)

(Also: I’m personally not a fan of `auto` for simple types like these, but that’s your call.)

Reply:
Nice catch, I think I missed the connection between that variable name and the Promises. Fixing that and cleaning up the auto to match the style of the rest of the file.